xdmod/README.md

```diff
@@ -62,39 +68,222 @@ The following information is needed by Open XDMoD:
 Optionally:
 
 - An image file containing the HPC center logo
+    - The width HPC center logo
 
 Also the following technical information:
 
 - The public url of Open XDMoD
 - Paths to installed dependencies (phantomjs)
 - MySQL connection information
+    - Host
+    - Port
+    - Admin Username
+    - Admin Password
+    - DB Username
+    - DB Password
+
+### Prerequisites used in this Tutorial
+
+- Name of the organization: `Tutorial` abbreviation: `hpcts`
+- information for each HPC resource
+    - Name: `hpc`
+    - Number of compute nodes: `2`
+    - Number of cores: `2`
+    - Timezone: `UTC`
+    - Whether it runs shared jobs: `no`
+- An image file containing the HPC center logo: `/srv/xdmod/small-logo.png`
+    - The width HPC center logo: `354`
+- The public url of Open XDMoD:  `https://localhost:4443`
+- Paths to installed dependencies (phantomjs):  `detected defaults`
+- MySQL connection information
+    - Host: `mysql`
+    - Port: `3306`
+    - Admin Username: `root`
+    - Admin Password: ` leave blank `
+    - DB Username: `xdmodapp`
+    - DB Password: `ofbatgorWep0`
```

Comment:
If you are installing the Job Performance module (as we are in this tutorial) you also need the access credentials for the MongoDB database